recipes/boost/all/conanfile.py

```diff
@@ -141,8 +141,12 @@ def configure(self):
                 if not self.options.get_safe('without_%s' % lib):
                     raise ConanInvalidConfiguration("Boost '%s' library requires multi threading" % lib)
 
+        if self.settings.compiler == "Visual Studio" and "MT" in self.settings.compiler.runtime and self.options.shared:
```

Comment:
@uilianries shouldn't `self.settings.compiler.runtime` be converted to a string?
```suggestion
        if self.settings.compiler == "Visual Studio" and "MT" in str(self.settings.compiler.runtime) and self.options.shared:
```

Reply:
Yes, it prevents cases where runtime is None. Thanks @madebr

Reply:
oh, I thought it was needed so it would match with `MT` and `MTd`.

Reply:
ah, not in this case, runtime is a SettingsItem, which is converted to str